### PR TITLE
[Support new order funnel - Part 1] Refactoring dependency handling

### PR DESCRIPTION
**Pull request by FredericEspiau**
# Support for new order funnel
This PR is part of the **Support new order funnel** project
### :nail_care: Refactor
Reorder the import and dependencies in the **global** and **Billing** module declaration

### :bug: Bugfix
Add missing dependency to **Billing** module declaration

## References
- MANAGER-2351

### CODE REVIEW

File: client/app/app.js [hunk 1]
```diff
@@ -1,99 +1,95 @@
...
+  .module('App', [
+    'Billing',
+    'chart.js',
+    'controllers',
+    dedicatedUniverseComponents,
+    'directives',
+    'filters',
+    'internationalPhoneNumber',
+    'Module.download',
+    ['eu', 'ca'].includes(WEBPACK_ENV.region) ? 'Module.exchange' : undefined,
```
Comment (antleblanc) [inline client/app/app.js]: Bear in mind that you potentially result with an `Error: [$injector:modulerr]`.

The  previous methods looks more safe.

**Comment by marie-j** [comment]
FYI you have two commits 8b61f358434a9847aa75bd4a88ccd692c789a36b and be515bfca3cae12546c98921a4d9df05159262bf that have the same commit message but two different types (`refactor` and `style`) 😉

**Comment by FredericEspiau** [comment]
@marie-j fixed !

File: client/app/app.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply client/app/app.js]: https://github.com/ovh-ux/ovh-manager-dedicated/pull/1017/files/93ba8d69e384c69c03743339f534e31f88fb4e88#diff-fa428ba3145392773209743143c2c38aR72 :)

**Pull request merged**